hexrd/unitcell.py

```diff
@@ -1258,6 +1314,8 @@ def a(self, val):
         self.ik = 1
         self.il = 1
         self.CalcMaxGIndex()
+        if(hasattr(self, 'numat')):
```

Comment:
You don't actually need these outer parentheses in `if` statements in python. Most people leave them out.